app/controllers/admin/deploy_groups_controller.rb

```diff
@@ -70,7 +72,7 @@ def build_kuber_cluster
     @deploy_group.build_cluster_deploy_group unless @deploy_group.cluster_deploy_group
   end
```

Comment:
@henders , which function? `build_cluster_deploy_group`?

What do you mean with not used anywhere?